sql/core/src/test/resources/tpcds-query-results/v2_7/q78.sql.out

```diff
@@ -0,0 +1,6 @@
+-- Automatically generated by TPCDSQueryTestSuite
+
+-- !query schema
+struct<ratio:double,store_qty:bigint,store_wholesale_cost:decimal(17,2),store_sales_price:decimal(17,2),other_chan_qty:bigint,other_chan_wholesale_cost:decimal(18,2),other_chan_sales_price:decimal(18,2)>
+-- !query output
+
```

Comment:
The same comment with https://github.com/apache/spark/pull/31886/files#r600489102